src/app/lib/analyticsUtils/index.js

```diff
@@ -229,6 +229,7 @@ export const getContentId = (assetType, pageData) => {
 
 export const getAtiUrl = (data = []) => {
   const cleanedValues = data
+    .filter(pageViewBeaconValues => pageViewBeaconValues)
```

Comment:
are you filtering out empty strings here? you could do

```suggestion
    .filter(Boolean)
```

Reply:
Filtering out null values here. `getRSSMarketingString(href, campaignType)` returns null when campaign is not `RSS`. Might change the return to be an empty array instead (based on your comment below regarding pattern) so this filter is no longer needed.